src/StripeWebhooksServiceProvider.php

```diff
@@ -15,16 +15,8 @@ public function boot()
             ], 'config');
         }
 
-        if (! class_exists('CreateStripeWebhookCallsTable')) {
-            $timestamp = date('Y_m_d_His', time());
-
-            $this->publishes([
-                __DIR__.'/../database/migrations/create_stripe_webhook_calls_table.php.stub' => database_path('migrations/'.$timestamp.'_create_stripe_webhook_calls_table.php'),
-            ], 'migrations');
-        }
-
         Route::macro('stripeWebhooks', function ($url) {
-            return Route::post($url, '\Spatie\StripeWebhooks\StripeWebhooksController');
+            return Route::post($url, StripeWebhooksController::class);
```

Comment:
I'm thinking this needs to be a string starting with `\` so also works with in apps where a namespace is set up in the service provider.